dist/index.js

```diff
@@ -19,9 +18,29 @@ export const avenirTracking = "0";
 export const avenirFamily = "'Avenir Next W01','Avenir Next W00','Avenir Next','Avenir','Helvetica Neue',sans-serif"; 
 export const codeTracking = "0"; 
 export const codeFamily = "'Consolas','Andale Mono','Lucida Console','Monaco',monospace"; 
-export const shadow1 = "0 0 12px 0 rgba(0,0,0,0.05)"; 
-export const shadow1Hover = "0 0 16px 0 rgba(0,0,0,0.1)"; 
-export const shadow1Press = "0 0 16px 0 rgba(0,0,0,0.2)"; 
-export const shadow2 = "0 0 16px 0 rgba(0,0,0,0.15)"; 
-export const shadow2Hover = "0 0 20px 0 rgba(0,0,0,0.2)"; 
-export const shadow2Press = "0 0 20px 0 rgba(0,0,0,0.3)"; 
+export const shadow1 = "0 0 8px 0 rgba(0,0,0,0.08)"; 
+export const shadow1Hover = "0 0 16px 0 rgba(0,0,0,0.08)"; 
+export const shadow1Press = "0 0 8px 0 rgba(0,0,0,0.16)"; 
+export const shadow2 = "0 0 16px 0 rgba(0,0,0,0.16)"; 
+export const shadow2Hover = "0 0 32px 0 rgba(0,0,0,0.16)"; 
+export const shadow2Press = "0 0 16px 0 rgba(0,0,0,0.32)"; 
+export const borderRadius = "3px"; 
```

Comment:
After running `npm start` or maybe `npm docs`, I did get a few changes that I didn't expect (such as the ones above to the shadow variables and the borderRadius addition.  I'm not sure if this is desired or not, it _does_ seem like the shadow variables are set to these new values elsewhere (`variables.scss`), so I left them in.